### DISCUSSION TITLE
Gboard

### DISCUSSION THREAD

**Discussion by No2515** [MustardChef/WSABuilds#222] [Q&A]
Install Setup asks to "Enable In settings."

Clicking the prompt takes me _Windows device settings_ instead of _Android_.

Gboard doesn't show up at all in Win device settings.

Unable to finish the setup. Any workaround?

**Comment by MustardChef** [comment]
Open WSA's Android Settings manually by typing the following into your browser's address bar: 

```
wsa://com.android.settings
```

**Reply by No2515** [reply]
I finished the install. Unfortunately I can't get the app to open. Opening the app manually takes me straight to gboard settings. 

I really wanted to replace windows OSK with gboard. Bummer

**Reply by MustardChef** [reply]
Well, even if GBoard can be set up, On-Screen Android keyboards cannot be used with WSA yet from what I can see.
